packages/contentstack-import/src/utils/backup-handler.ts

```diff
@@ -37,7 +41,7 @@ export default async function backupHandler(importConfig: ImportConfig): Promise
 
   if (backupDirPath) {
     cliux.print('Copying content to the backup directory...');
-
+    await copyMapperFolder(importConfig, backupDirPath);
```

Comment:
"copyMapperFolder" this can be removed for now

Reply:
Please keep this code if you are hiding the mapper-dir flag

Reply:
Removed